veggieseasons/lib/main.dart

```diff
@@ -14,9 +15,12 @@ void main() {
   ]);
 
   runApp(
-    CupertinoApp(
-      color: Styles.appBackground,
-      home: HomeScreen(),
+    ScopedModel<AppState>(
+      model: AppState(),
+      child: CupertinoApp(
+        color: Color(0xffd0d0d0),
```

Comment:
This could be `Colors.gray[350]` (which is `0xFFD6D6D6`).

Reply:
`Colors` is part of Material, so I can't use it. I made a constant for this in `Styles` that I wasn't using, though, so thanks for catching it.